moncli/entities/column_value/simple.py

```diff
@@ -13,7 +13,47 @@ class DateValue(ComplexNullValue):
 
 class DropdownValue(ComplexNullValue):
     """A dropdown column value."""
-    pass
+    native_type = list
+    native_default = []
+
+    def _convert(self, value):
+        labels = self.settings['labels']
+        ids = value['ids']
+        label_list = []
+
+        for item in labels:
+            if item['id'] in ids:
+                label_list.append(item['name'])   
+        return label_list
+
+    def _format(self):
```

Comment:
I forgot to add this to the requirements, but we will also need to remove any duplicate format values.